typescript/ws-protocol/src/types.ts

```diff
@@ -2,6 +2,7 @@ export enum BinaryOpcode {
   MESSAGE_DATA = 1,
   TIME = 2,
   SERVICE_CALL_RESPONSE = 3,
+  ASSET = 4,
```

Comment:
```suggestion
  FETCH_ASSET_RESPONSE = 4,
```

Maybe more suited, now that the response can also be a error response